apps/landingPage/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@kiwicom/margarita-landing-page",
+  "private": true,
+  "version": "0.1.0",
+  "dependencies": {
```

Comment:
I realized that there are few gatsby plugins which are not used anywhere, can you check and clean them.